addons/mail/static/src/components/message_action_list/message_action_list.xml

```diff
@@ -3,7 +3,7 @@
     <t t-name="mail.MessageActionList" owl="1">
         <div class="o_MessageActionList d-flex" t-on-click="messageActionList and messageActionList.onClick">
             <t t-if="messageActionList">
-                <Popover class="o_MessageActionList_action o_MessageActionList_actionReaction p-2 fa fa-lg fa-smile-o" position="'top'" titleAttribute="ADD_A_REACTION" t-on-o-emoji-selection="messageActionList.onEmojiSelection" t-on-o-popover-closed="messageActionList.onReactionPopoverClosed" t-on-o-popover-opened="messageActionList.onReactionPopoverOpened" t-ref="reactionPopover">
+                <Popover t-if="messageActionList.message.canStarBeToggled" class="o_MessageActionList_action o_MessageActionList_actionReaction p-2 fa fa-lg fa-smile-o" position="'top'" titleAttribute="ADD_A_REACTION" t-on-o-emoji-selection="messageActionList.onEmojiSelection" t-on-o-popover-closed="messageActionList.onReactionPopoverClosed" t-on-o-popover-opened="messageActionList.onReactionPopoverOpened" t-ref="reactionPopover">
```

Comment:
Hello, please make a new field for reactions, the condition for star is not correct here (in particular guest should still be able to star other messages).